### PR TITLE
WIP: Phone booth blog post

### PR DESCRIPTION
**Pull request by IshaDakota**

### CODE REVIEW

**Comment by RachelBaram** [comment]
@IshaDakota Cleaning up the queue... what is the status of this? Should this be closed?

**Comment by IshaDakota** [comment]
@RachelBaram @kazabooboo look over here!

File: app/_posts/2016-08-15-phone-booth.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Talking from a Phone Booth: Strategies for Working Remotely"
+tags:
+  - The Gizra Way
+permalink: "/content/talking-from-a-phone-booth"
+layout: post
+author: kazabooboo
+image: /assets/images/posts/elm-electron-build/thumb.jpg
+description:
```
Comment (RachelBaram) [inline app/_posts/2016-08-15-phone-booth.md]: @kazabooboo Can you please write a description. It should be less than 160 characters.

File: app/_posts/2016-08-15-phone-booth.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Out of the total 20+ Gizra team, there has been increasingly more of us working remotely. Some of us are in Israel but live far away (well, relativity) from the city of Tel Aviv, and others, are on different continents altogether. It has been a real challenge for all of us -  those that work remotely, and for those that work from desks at the main office - how not to be strangers to each other, and how to feel that we are part of the same team, sharing the same culture, visions, values, and even internal jokes. How Gizra has or is planning to address this challenge, we will keep for a future post. 
+
+I would like to go back to the phone booth.
+
+## Co-working space practices, The Gizra Way 
+
+After few months here,  I came up with the Gizra Way [link] for co-working space etiquette. This is not one of those “do not steal someone else's food from the fridge" or "don't bring stinky meals” lists. If you don’t know that, then you should look for a basic etiquette class. Do those still exist? Anyways...
```
Comment (RachelBaram) [inline app/_posts/2016-08-15-phone-booth.md]: @kazabooboo Were you planning on putting something in [link]?

File: app/_posts/2016-08-15-phone-booth.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+1. <b>Time box your day (and have your headphones on)</b> : When working by yourself without anyone around to interrupt you, it’s easy to be carried away and not notice that you have been researching hotels for an upcoming trip to DC for more than hour now. Fix a time period for every task in order to put boundaries on the task so that you don’t need to skip lunch (see #2).  I use a timer that alerts me every 25 min, and encourages me to take quick mental breaks. In Gizra, we use our own grassroots system to log in project contributions hours, we call it Productivity [link to module]. Why the headphones? This is the open space equivalent for “do not disturb” sign. If your co-space neighbors do not know that, move to a different desk or different space altogether.
```
Comment (RachelBaram) [inline app/_posts/2016-08-15-phone-booth.md]: @kazabooboo Don't forget the link to productivity.

Comment (amitaibu) [inline app/_posts/2016-08-15-phone-booth.md]: No need to use HTML tags like `<b>` , you can use markdown =>`_Time box .._` or `__Time box ..__`  which results with: _Time box .._ / __Time box ..__

File: app/_posts/2016-08-15-phone-booth.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+## Have we mentioned our US office before? 
+
+Gizra expanded to what some people say is the greatest city in the world, operating from a co-working space in Manhattan. A space where the booth I speak about is located. The booth provides privacy and is also the right place to go, in such an environment, when taking a phone call or meeting. I doubt that the guy I am sharing a power strip with cares to hear me speaking about Drupal modules, NYC tax breaks for startups, or Brice’s daughter’s ear infection. 
+
+Out of the total 20+ Gizra team, there has been increasingly more of us working remotely. Some of us are in Israel but live far away (well, relativity) from the city of Tel Aviv, and others, are on different continents altogether. It has been a real challenge for all of us -  those that work remotely, and for those that work from desks at the main office - how not to be strangers to each other, and how to feel that we are part of the same team, sharing the same culture, visions, values, and even internal jokes. How Gizra has or is planning to address this challenge, we will keep for a future post.
```
Comment (amitaibu) [inline app/_posts/2016-08-15-phone-booth.md]: > How Gizra has or is planning to address this challenge, we will keep for a future post. 

I'd remove this. Future posts are a promise I've yet to see fulfield by anyone

File: app/_posts/2016-08-15-phone-booth.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Talking from a Phone Booth: Strategies for Working Remotely"
+tags:
+  - The Gizra Way
+permalink: "/content/talking-from-a-phone-booth"
+layout: post
+author: kazabooboo
+image: /assets/images/posts/elm-electron-build/thumb.jpg
```
Comment (RachelBaram) [inline app/_posts/2016-08-15-phone-booth.md]: maybe a 'Doctor Who' phone booth? ;)

File: app/_posts/2016-08-15-phone-booth.md (hunk 4 above)
Comment (RachelBaram) [inline app/_posts/2016-08-15-phone-booth.md]: done

**Comment by RachelBaram** [comment]
@kazabooboo After you go over the various notes (don't forget to expand and look at those that are 'outdated') and make the required changes, please let me know and I will give it a final look over.

File: app/_posts/2016-08-15-phone-booth.md (hunk 5 above)
Comment (IshaDakota) [inline app/_posts/2016-08-15-phone-booth.md]: Old american phone?

File: app/_posts/2016-08-15-phone-booth.md (hunk 1 above)
Comment (kazabooboo) [inline app/_posts/2016-08-15-phone-booth.md]: @RachelBaram - looks good. From some reason I can not edit now. (I guess I messed up that PR real good.)
Suggested Description: "Working remotely surrounded by strangers is a challenge which can become opportunity by following few rules of thumb" feel free to change it

File: app/_posts/2016-08-15-phone-booth.md (hunk 5 above)
Comment (kazabooboo) [inline app/_posts/2016-08-15-phone-booth.md]: I like one of the pictures [here](https://www.flickr.com/search/?sort=date-taken-desc&safe_search=1&tags=oldphones&user_id=72024882%40N00&view_all=1).  (They are under CC so I am assuming that we can use them).

**Comment by RachelBaram** [comment]
@kazabooboo It would be great to publish this. Waiting for your changes.

**Comment by kazabooboo** [comment]
@RachelBaram - I can't edit the file from some reason. @IshaDakota Any idea why?

**Comment by kazabooboo** [comment]
![img_5273](https://cloud.githubusercontent.com/assets/16226208/20270167/0c621c52-aa54-11e6-8218-b57b0a0472f7.JPG
This is how Gizra's USA NYC desk looks like

File: app/_posts/2016-08-15-phone-booth.md (hunk 5 above)
Comment (IshaDakota) [inline app/_posts/2016-08-15-phone-booth.md]: we actually can't use... license is for non-commercial use.

Comment (kazabooboo) [inline app/_posts/2016-08-15-phone-booth.md]: 4olnw

**Comment by IshaDakota** [comment]
@kazabooboo publlsh?

**Comment by kazabooboo** [comment]
@IshaDakota As long that it is  inline with our content strategy

**Comment by IshaDakota** [comment]
@kazabooboo we'll make an exception this time ;)

**Comment by IshaDakota** [comment]
merged and deployed.

**Pull request merged**